### PR TITLE
Add support for cross arch for fluent-bit-plugin-loki image

### LINKED ISSUE #6738: grafana/fluent-bit-plugin-loki is not available in arm64
**Is your feature request related to a problem? Please describe.**
I cannot use fluentbit loki in Fargate `arm64` instances, since this docker image is only available for `amd64`

**Describe the solution you'd like**
Instead of using `version-arch` (example: `grafana/fluent-bit-plugin-loki:2.5.0-amd64`), ~use~ also the release version without arch and compile this image for both architectures

**Describe alternatives you've considered**
Right now this forces me to compile and deploy my ecs containers in `amd64`.
I know how to do this cross-compile in github-actions, however I'm not familiar with drone builds

**Additional context**
_none_

### LINKED ISSUE #9932: fluentbit plugin latest docker image not working
**Describe the bug**

The latest docker image from docker hub (`grafana/fluent-bit-plugin-loki:main`) fails to start.
Reverting to `grafana/fluent-bit-plugin-loki:main-9e19ff0-amd64` seems to work fine.

**To Reproduce**

Just start the image with `docker run`

**Expected behavior**

The image starting normally.

**Environment:**
 - Infrastructure: AWS ECS, docker
 - Deployment tool: terraform

**Screenshots, Promtail config, or terminal output**
```
[error] [plugin] error loading proxy plugin: /fluent-bit/bin/out_grafana_loki.so
[proxy] error opening plugin /fluent-bit/bin/out_grafana_loki.so: '/lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /fluent-bit/bin/out_grafana_loki.so)'
```

### PR DESCRIPTION
**Pull request by ebosveld**
**What this PR does / why we need it**:
Currently the `fluent-bit-plugin-loki` image is only build and published for amd64. With more use of arm-based processors, it would be good to add support for those as well. 

I've build the image using `docker buildx` on my M1 Mac and pushed it to Dockerhub, should you wish to try it first:
https://hub.docker.com/r/ebosveld/fluent-bit-plugin-loki/

On AWS ECS Graviton based images it works well with the above mentioned image.

I've also fixed #9932 as it needed the full bullseye support.

**Which issue(s) this PR fixes**:
Fixes #6738
Fixes #9932

**Special notes for your reviewer**:

**Checklist**
- [x] Reviewed the [`CONTRIBUTING.md`](https://github.com/grafana/loki/blob/main/CONTRIBUTING.md) guide (**required**)
- [ ] Documentation added
- [ ] Tests updated
- [ ] `CHANGELOG.md` updated
  - [ ] If the change is worth mentioning in the release notes, add `add-to-release-notes` label
- [ ] Changes that require user attention or interaction to upgrade are documented in `docs/sources/upgrading/_index.md`
- [ ] For Helm chart changes bump the Helm chart version in `production/helm/loki/Chart.yaml` and update `production/helm/loki/CHANGELOG.md` and `production/helm/loki/README.md`. [Example PR](https://github.com/grafana/loki/commit/d10549e3ece02120974929894ee333d07755d213)

### CODE REVIEW

**Comment by ebosveld** [comment]
Not sure who to tag, but it says @slim-bean is code owner. My build says it's blocked: "Build is blocked, please, contact repo admin in order to proceed" - can someone have a look to give it a little kick so it's running again?

File: .drone/drone.yml [hunk 1]
```diff
@@ -1014,6 +1014,112 @@ trigger:
 depends_on:
 - check
 kind: pipeline
+name: fluent-bit-arm64
```
[the remark this answers is not in the document]
Comment (ebosveld) [reply .drone/drone.yml]: Ah yeah, I understand now @MichelHollands. I've made the changes there.

**Comment by MasslessParticle** [comment]
I've made this PR against your branch that contains the generated file. That should help get this through.

https://github.com/ebosveld/loki/pull/1

**Comment by MichelHollands** [comment]
@ebosveld Something went wrong there but I don't fully understand why. The CI fails with `linter: invalid or unknown pipeline dependency`. I created a new PR which looks the same but does work: https://github.com/grafana/loki/pull/10081. If it's OK with you I'll merge that one.

**Comment by MichelHollands** [comment]
@ebosveld update: it seems that there were some missing drone.yml lines.

**Comment by MichelHollands** [comment]
Superseded by https://github.com/grafana/loki/pull/10081

**Comment by MasslessParticle** [comment]
@ebosveld Thanks for your contribution. Sorry for the confusion! I've merged https://github.com/grafana/loki/pull/10081 in liu of this PR.

**Pull request closed without merging**